### ISSUE TITLE
Allocation Change Request: "Add new attribute"

### ISSUE THREAD

**Issue by mdzik** [ubccr/coldfront#400]
Some attributes are unique, others are not (for example IP addresses specified as attributes by PI in license allocations). I would make sens to allow not only change but to request new attribute added.

I was thinking, to extend change request to allow that if:
- attribute is changeable
- attribute in not unique
- allocation already have at least one of that type
- maybe also add some flag, per analogy to 'is_changeable'?

This will require slight change in the model to accommodate data for non-existing attribute before approval. Is this ok your cases?

**Comment by aebruno** [comment]
@mkzia Yes, this all sounds good to me. @dsajdak do you have issues/suggestions to the above changes with respect to our workflow?
